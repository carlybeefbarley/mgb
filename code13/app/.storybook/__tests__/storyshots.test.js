// Jest Snapshot testing for stories
// https://github.com/storybooks/storybook/tree/master/addons/storyshots
import initStoryshots from '@storybook/addon-storyshots'

initStoryshots()
