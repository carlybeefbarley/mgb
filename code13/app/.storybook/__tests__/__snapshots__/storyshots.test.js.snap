// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots __Examples__ Actions 1`] = `
<div
  className="ui text container"
>
  <p>
    Stories can log actions. Open the "Action Logger" panel and click the button.
  </p>
  <button
    className="ui button"
    disabled={undefined}
    onClick={[Function]}
    role="button"
    tabIndex={undefined}
  >
    Log Action
  </button>
</div>
`;

exports[`Storyshots __Examples__ Knobs 1`] = `
<div
  className="ui text container"
>
  <p>
    Knobs make your story interactive. It is a good practice to make a "knob" for each prop you are showing the story for.
  </p>
  <p>
    Open the "Knobs" panel and try it out.
  </p>
  <div
    className="ui segment"
  >
    <button
      className="ui button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      Click Me
    </button>
  </div>
</div>
`;

exports[`Storyshots __Examples__ Links 1`] = `
<div
  className="ui text container"
>
  <p>
    Stories can link to other stories.
  </p>
  <button
    className="ui icon primary left labeled button"
    disabled={undefined}
    onClick={[Function]}
    role="button"
    tabIndex={undefined}
  >
    <i
      aria-hidden="true"
      className="level up icon"
    />
    Go to 'Knobs'
  </button>
</div>
`;

exports[`Storyshots SupportedBrowsers Default 1`] = `
<div
  className="ui red inverted center aligned segment"
  style={
    Object {
      "margin": 0,
    }
  }
>
  <p>
    <i
      aria-hidden="true"
      className="warning sign inverted icon"
    />
    Your browser doesn't support newer features we use, please upgrade:
  </p>
  <a
    className="ui tiny basic compact inverted button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="chrome icon"
    />
    Chrome
  </a>
  <span>
     
  </span>
  <a
    className="ui red tiny compact button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="firefox icon"
    />
    Firefox
  </a>
  <span>
     
  </span>
  <a
    className="ui red tiny compact button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="safari icon"
    />
    Safari
  </a>
</div>
`;

exports[`Storyshots SupportedBrowsers Recommended Buttons 1`] = `
<div
  className="ui red inverted center aligned segment"
  style={
    Object {
      "margin": 0,
    }
  }
>
  <p>
    <i
      aria-hidden="true"
      className="warning sign inverted icon"
    />
    Your browser doesn't support newer features we use, please upgrade:
  </p>
  <a
    className="ui tiny basic compact inverted button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="asterisk icon"
    />
    Recommended
  </a>
  <span>
     
  </span>
  <a
    className="ui tiny basic compact inverted button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="asterisk icon"
    />
    Also recommended
  </a>
  <span>
     
  </span>
  <a
    className="ui red tiny compact button"
    disabled={undefined}
    href="javascript:;"
    onClick={[Function]}
    rel="noreferrer nofollow"
    role="button"
    tabIndex={undefined}
    target="_blank"
  >
    <i
      aria-hidden="true"
      className="asterisk icon"
    />
    Not Recommended
  </a>
</div>
`;
